### DISCUSSION TITLE
Embed online playground into helix-editor.com

### DISCUSSION THREAD

**Discussion by tomgroenwoldt** [helix-editor/helix#7562] [Ideas] [answered]
I recently worked on an [online playground for helix](https://github.com/tomgroenwoldt/helix-editor-playground) and asked myself if it would be a cool feature to embed something similar into the https://helix-editor.com homepage. This way a potential new user would be able to dive right into the editor experience.

I would be happy to provide the backend for this but then again I guess this would raise some security concerns.
What do you think about this?

**Comment by the-mikedavis** [comment]
I saw the post and I think it's quite cool. I'm hesitant about the security parts and of maintaining hardware / infrastructure in general though. IMO it's not worth the extra maintenance burden for us to make this "official"

**Comment by archseer** [comment, marked as answer]
An xterm.js (pepper has a great demo like that https://vamolessa.github.io/pepper/web/) or WASM based port is a potential experiment/long-term goal for me. Then we'd be able to host it without requiring a server to back it
